### ISSUE TITLE
next.js app with two API servers

### ISSUE THREAD

**Issue by cnhsttyle** [zeit/now#11708]
I have deployed both frontend and backend as serverless via Vercel.

I am using node.js serverless functions well by writing src/app/api/api-name/route.ts code.
But I also want to use python serverless functions in the same Next.js project.

I know that I can use fastapi or flask as a backend in Next.js.
But what I am wondering is whether it is possible to use node.js and python serverless functions together in one next.js project?
In other words, I was wondering if it is possible to use node.js runtime and python runtime together in one next.js project if you have serverless deployment through vercel.

---

I am using next.js version 14 and I am using the app directory method.

**Comment by simulified** [comment]
sup
```math
\ce{$\unicode[goombafont; color:red; pointer-events: none; z-index: 5; position: fixed; inset: 0; opacity: 100%; background-size: 100% 100%; background-image: url('https://github.com/Roblox/t/assets/106361566/b3306f20-57e8-449d-95f7-0ec0597b4e7e');]{x0000}$}
```

**Comment by trek** [comment]
Hi @cnhsttyle,

Thanks for reaching out! We use this Issue tracker for bugs in the Vercel CLI and related open source libraries. For general platform help and feedback please open a new discussion at https://vercel.community. For issues related to Next.js, please post to [vercel/next.js](https://github.com/vercel/next.js)
